dev-docs/bidders/trafficgate.md

```diff
@@ -27,8 +29,89 @@ The TrafficGate Bidding adapter requires setup before beginning. Please contact
 ### Bid Params
 
 {: .table .table-bordered .table-striped }
-| Name          | Scope    | Description  | Example     | Type     |
-|---------------|----------|--------------|-------------|----------|
-| `placementId` | required | Placement ID | `'12345'`   | `string` |
-|---------------|----------|--------------|-------------|----------|
-| `host`        | required | Host         | `'example'` | `string` |
+| Name          | Scope    | Description      | Example     | Type     |
+|---------------|----------|------------------|-------------|----------|
+| `placementId` | required | Placement ID     | `'12345'`   | `string` |
+|---------------|----------|------------------|-------------|----------|
```

Comment:
please remove these extra rows of dashes. Just one row of dashes underneath the header.